src/webhook_dispatches.js

```diff
@@ -0,0 +1,113 @@
+ *
+ * const apifyClient = new ApifyClient({
+ *  userId: 'jklnDMNKLekk',
+ *  token: 'SNjkeiuoeD443lpod68dk',
+ * });
+ *
+ * // Awaited promise
+ * try {
+ *      const dispatchesList = await apifyClient.webhookDispatches.listWebhookDispatches({});
+ *      // Do something with the list ...
+ * } catch (err) {
+ *      // Do something with error ...
+ * }
+ *
+ * // Promise
+ * apifyClient.webhookDispatches.listWebhookDispatches({})
+ * .then((tasksList) => {
+ *      // Do something with the list ...
+ * })
+ * .catch((err) => {
+ *      // Do something with error ...
+ * });
+ *
+ * // Callback
+ * apifyClient.webhookDispatches.listWebhookDispatches({}, (err, tasksList) => {
+ *      // Do something with error or list ...
+ * });
+ * ```
+ * @namespace webhookDispatches
+ */
+export const BASE_PATH = '/v2/webhook-dispatches';
+
+
+export default {
+    /**
+     * Gets list of webhook dispatches.
+     * @description By default, the objects are sorted by the startedAt field in ascending order,
+     * therefore you can use pagination to incrementally fetch all dispatches.
+     * To sort them in descending order, use desc: `true` parameter.
+     * The endpoint supports pagination using limit and offset parameters and it will not return more than 1000 array elements.
+     * @memberof ApifyClient.webhookDispatches
+     * @instance
+     * @param {Object} options
+     * @param options.token
+     * @param {Number} [options.offset=0] - Number of array elements that should be skipped at the start.
+     * @param {Number} [options.limit=1000] - Maximum number of array elements to return.
+     * @param {Boolean} [options.desc] - If `true` then the objects are sorted by the startedAd field in descending order.
+     * @param callback
+     * @returns {PaginationList}
+     */
+    listWebhookDispatches: (requestPromise, options) => {
+        const { baseUrl, token, offset, limit, desc } = options;
+
+        checkParamOrThrow(baseUrl, 'baseUrl', 'String');
+        checkParamOrThrow(token, 'token', 'String');
+        checkParamOrThrow(limit, 'limit', 'Maybe Number');
+        checkParamOrThrow(offset, 'offset', 'Maybe Number');
+        checkParamOrThrow(desc, 'desc', 'Maybe Boolean');
+
+        const query = { token };
+
+        if (limit) query.limit = limit;
+        if (offset) query.offset = offset;
+        if (desc) query.desc = 1;
+
+        return requestPromise({
+            url: `${baseUrl}${BASE_PATH}`,
+            json: true,
+            method: 'GET',
+            qs: query,
+        })
+            .then(pluckData)
+            .then(parseDateFields);
+    },
+    /**
+     * Gets webhook dispatch.
+     * @memberof ApifyClient.webhookDispatches
+     * @instance
+     * @param {Object} options
+     * @param options.token
+     * @param option.webhookDispatchId - Webhook dispatch ID
+     * @param callback
+     * @returns {WebhookDispatch}
+     */
+    getWebhookDispatch: (requestPromise, options) => {
```

Comment:
You have `listDispatches` function in `webhooks` and `listWebhookDispatches ` and `getWebhookDispatch` here. This should be consistent - I'd call the functions here `listDispatches` and `getDispatch`.

Reply:
I followed the API structure:
/webhook-dispatches -> webhookDispatches.listWebhookDispatches()
/webhook-dispatches/[id] -> webhookDispatches.getWebhookDispatch()
/webhooks/[id]/dispatches -> webhooks.listDispatches()

It is true that we can simplify it to 
/webhook-dispatches -> webhookDispatches.listDispatches()
/webhook-dispatches/[id] -> webhookDispatches.getDispatch()